langserve/playground/src/components/feedback/CorrectnessFeedback.tsx

```diff
@@ -0,0 +1,104 @@
+import ThumbsUpIcon from "../../assets/ThumbsUpIcon.svg?react";
+import ThumbsDownIcon from "../../assets/ThumbsDownIcon.svg?react";
+import CircleSpinIcon from "../../assets/CircleSpinIcon.svg?react";
+import { resolveApiUrl } from "../../utils/url";
+import { useEffect, useRef, useState } from "react";
+import { cn } from "../../utils/cn";
+
+export function CorrectnessFeedback(props: { runId?: string }) {
```

Comment:
I don't have any frontend experience so please take my approval on this with a grain of salt :)